src/imports/logger/ErrorsTransport.js

```diff
@@ -63,7 +53,13 @@ export default class ErrorsTransport extends Transport {
         })
       })
 
-      Sentry.captureException(errorToPassIntoLog)
+      try {
+        errorObj.message = `${generalMessage}: ${originalErrMsg}`
+        Sentry.captureException(errorObj)
+        await Sentry.flush()
+      } finally {
+        errorObj.message = originalErrMsg
+      }
```

Comment:
line 5
```javascript
import { assign, forEach } from 'lodash'
```

```suggestion
        errorObj.message = `${generalMessage}: ${message}`
        Sentry.captureException(errorObj)
        await Sentry.flush().finally(() => assign(errroObj, { message }))         
```
(actually the flush should be try..catched)